### PR TITLE
docs (ldap) service directory mapping workaround

### PR DESCRIPTION
**Pull request by GayleNeumann**
-- Created ldap_service_directory mapping workaround according to FTI-1758. 
-- _include used on the following doc topics:
* https://docs.konghq.com/hub/kong-inc/ldap-auth-advanced/
* https://docs.konghq.com/hub/kong-inc/ldap-auth/
* https://docs.konghq.com/enterprise/2.1.x/kong-manager/authentication/ldap/
Added as a known issue entry to release notes and changelog:
* https://docs.konghq.com/enterprise/2.1.x/release-notes/
* https://docs.konghq.com/enterprise/changelog/

See preview at https://deploy-preview-2310--kongdocs.netlify.app/hub/kong-inc/ldap-auth/#using-service-directory-mapping-on-the-cli

### CODE REVIEW

File: app/enterprise/2.1.x/kong-manager/authentication/ldap.md [hunk 1]
```diff
@@ -59,3 +61,9 @@ Learn more about these properties in [Session Security in Kong Manager](/enterpr
 After starting Kong with the desired configuration, you can create new *Admins*
 whose usernames match those in the AD. Those users will then be able to accept
 invitations to join Kong Manager and log in with their LDAP credentials.
+
+### Using Service Directory Mapping on the CLI
```
Comment (tlblessing) [inline app/enterprise/2.1.x/kong-manager/authentication/ldap.md:65]: There are double headings for this, so you'll need to remove the heading from the include, or remove this heading here. 
image

**Review by tlblessing — CHANGES_REQUESTED** [review]
fix double titles due to include

File: app/enterprise/2.1.x/kong-manager/authentication/ldap.md (hunk 1 above)
Comment (GayleNeumann) [reply app/enterprise/2.1.x/kong-manager/authentication/ldap.md:65]: Thanks, Terry. This was still a WIP, which I should have noted on the PR. Appreciate the input.

**Comment by tlblessing** [comment]
@GayleNeumann I'm seeing this error in the build:

`[11:39:37]   Liquid Exception: Invalid syntax for include tag. File contains invalid characters or sequences: /md//ldap/ldap-service-directory-mapping.md Valid syntax: {% include file.ext param='value' param2='value' %} in /srv/jekyll/app/_hub/kong-inc/ldap-auth-advanced/index.md`

I have only tried using an include in the non-plugin hub area of the docs. I'll keep troubleshooting.

File: app/enterprise/2.1.x/kong-manager/authentication/ldap.md (hunk 1 above)
Comment (GayleNeumann) [reply app/enterprise/2.1.x/kong-manager/authentication/ldap.md:65]: I removed the heading from the include, so this is resolved. See https://deploy-preview-2310--kongdocs.netlify.app/hub/kong-inc/ldap-auth/#using-service-directory-mapping-on-the-cli

**Review by tlblessing — APPROVED** [review]
Checked include in following locations:

- https://deploy-preview-2310--kongdocs.netlify.app/hub/kong-inc/ldap-auth/#using-service-directory-mapping-on-the-cli
- https://deploy-preview-2310--kongdocs.netlify.app/enterprise/2.1.x/kong-manager/authentication/ldap/
- https://deploy-preview-2310--kongdocs.netlify.app/hub/kong-inc/ldap-auth-advanced/#using-service-directory-mapping-on-the-cli
- https://deploy-preview-2310--kongdocs.netlify.app/enterprise/2.1.x/kong-manager/authentication/ldap/
- https://deploy-preview-2310--kongdocs.netlify.app/enterprise/2.1.x/release-notes/#known-issues-and-workarounds
- https://deploy-preview-2310--kongdocs.netlify.app/enterprise/changelog/#known-issues-and-workarounds

LGTM

**Pull request merged**